### ISSUE TITLE
Failed to initialize REPL environment

### ISSUE THREAD

**Issue by mricher-git** [ManlyMarco/RuntimeUnityEditor#68]
Getting the following issue in Railroader:
[RuntimeUnityEditor] [Warning] Failed to initialize REPL environment - The type initializer for 'Mono.CSharp.SkipVisibilityExt' threw an exception.

Seems [https://github.com/5rog/Explorer](url) has the same issue with mcs and commented out the constructor. I tried commented it out, REPL loads but then I get:

error CS0060: Inconsistent accessibility: base class `RuntimeUnityEditor.Core.REPL.REPL' is less accessible than class `<InteractiveExpressionClass 4>'

Unsure if the two are related.

**Comment by ManlyMarco** [comment]
Yes, the class in the error is specifically to fix accessibility issues like the one you've got.

**Comment by ManlyMarco** [comment]
Did you try making `RuntimeUnityEditor.Core.REPL.REPL` public?

**Comment by mricher-git** [comment]
Isn't it already?
```
namespace RuntimeUnityEditor.Core.REPL
{
    /// <summary>
    /// C# REPL environment. Everything in here can be called directly in the REPL console.
    /// </summary>
    public class REPL : InteractiveBase
    ```

**Comment by ManlyMarco** [comment]
Oh, that's right, no idea then. I don't think OS has any effect on it, the game must be set up differently.

**Comment by mricher-git** [comment]
In 5.3 UMM flavor there is both a RuntimeUnityEditor.UMM.dll and a RuntimeUnityEditor.Core.dll.

From dnSpy, the UMM dll:
```
namespace RuntimeUnityEditor.Core.REPL
{
	// Token: 0x0200002A RID: 42
	internal class REPL : InteractiveBase
	{
```

From the .Core.dll:
```
namespace RuntimeUnityEditor.Core.REPL
{
	/// <summary>
	/// C# REPL environment. Everything in here can be called directly in the REPL console.
	/// </summary>
	// Token: 0x0200001F RID: 31
	public class REPL : InteractiveBase
	{
```

**Comment by ManlyMarco** [comment]
So this commit can be safely reverted now? https://github.com/ManlyMarco/RuntimeUnityEditor/commit/55252e7ab8b7dda0c4123f987c1829316f62128a
